lib/openshift/deployment_config.rb

```diff
@@ -113,6 +112,11 @@ def selector(user: nil, cached: true, quiet: false)
         dig("spec", "selector")
     end
 
+    def containers(user: nil, cached: false, quiet: false)
+      raw_resource(user: user, cached: cached, quiet: quiet).
+        dig("spec", "template", "spec", "containers")
```

Comment:
Could you test out this change ?

Reply:
@liangxia the change works, here is the case using the function: https://mastern-jenkins-csb-openshift-qe.apps.ocp4.prod.psi.redhat.com/job/Runner-v3/321092/console